lambda/onomondo/getAllUsedSimsOnomondo.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchAndValidate } from '../fetchAndValidate.js'
+import { Type, type Static } from '@sinclair/typebox'
+import { fromEnv } from '@nordicsemiconductor/from-env'
+
+const { APIKEY_ONOMONDO } = fromEnv({
+	APIKEY_ONOMONDO: 'APIKEY_ONOMONDO',
+})(process.env)
+
+export const getAllUsedSimsOnomondo = async (): Promise<Array<string>> => {
+	const iccidArray: Array<string> = []
```

Comment:
```suggestion
	const iccids: Array<string> = []
```

style: do not use technical details in variable names.